app/js/components/questions/new/NewQuestionPage.react.js

```diff
@@ -60,8 +62,16 @@ const NewQuestionPage = ({
                          fourteen wide tablet \
                          fourteen wide computer column">
 
-                         <TinyMCE
-                           content=""
+                         <textarea
+                           content={question.content}
+                           data-id="content"
+                           onChange={event => {
+                             onUpdateQuestionState('content',
+                             event.target.value);
+                           }} />
+
+                         {/*<TinyMCE
```

Comment:
Let's endeavour to remove commented out code. We can always go back to the git history to retrieve them if ew need to